src/lhm.net/AtomicSwitcher.cs

```diff
@@ -15,21 +16,38 @@ public AtomicSwitcher(TableMigration migration, ILhmConnection connection)
             _connection = connection;
         }
 
-
         public void Run()
         {
             Logger.Info(string.Format("Renaming origin table {0} to archive table {1}", _migration.Origin.Name, _migration.ArchiveName));
 
-            var sql = string.Format(@"DECLARE @TranName VARCHAR(20);
-                                      SELECT @TranName = 'LHM_Rename_Table';
-                                      BEGIN TRANSACTION @TranName;
+            var sql = $@"DECLARE @TranName VARCHAR(20);
+                        SELECT @TranName = 'LHM_Rename_Table';
+                        BEGIN TRANSACTION @TranName;
 
-                                        exec sp_rename [{0}], [{1}];
-                                        exec sp_rename [{2}], [{0}] 
+                        exec sp_rename [{_migration.Origin.Name}], [{_migration.ArchiveName}];
+                        exec sp_rename [{_migration.Destination.Name}], [{_migration.Origin.Name}] 
 
-                                      COMMIT TRANSACTION @TranName;", _migration.Origin.Name, _migration.ArchiveName, _migration.Destination.Name);
+                        COMMIT TRANSACTION @TranName;";
             
             _connection.Execute(sql);
+
+            sql = $@"SELECT obj.name AS FK_NAME
```

Comment:
I put this bit in a named method to make it clear what its doing.  RemoveForigenKeysFromArchiveTable